integration/commands_administration_compat_test.go

```diff
@@ -85,11 +84,8 @@ func TestCommandsAdministrationCompatCollStatsWithScale(t *testing.T) {
 				targetErr = UnsetRaw(t, targetErr)
 				compatErr = UnsetRaw(t, compatErr)
 
-				// TODO https://github.com/FerretDB/FerretDB/issues/2545
 				if tc.altMessage != "" {
-					var expectedErr mongo.CommandError
-					require.ErrorAs(t, compatErr, &expectedErr)
-					AssertEqualAltError(t, expectedErr, tc.altMessage, targetErr)
+					AssertMatchesCommandError(t, compatErr, targetErr)
```

Comment:
Note that `tc.altMessage` is not used there.

If then intention was to do a part of #2345 too, the I think the whole `if-else` could be replaced by a single call to `AssertMatchesCommandError`

Reply:
Replaced if-else with `AssertMatchesCommandError`. I also removed `tc.altMessage` as well since it was no longer used. So this issue also closes https://github.com/FerretDB/FerretDB/issues/2345